components.go

```diff
@@ -830,8 +823,7 @@ func (calendar *Calendar) AddVAlarm(e *VAlarm) {
 func (calendar *Calendar) Alarms() []*VAlarm {
 	var r []*VAlarm
 	for i := range calendar.Components {
-		switch alarm := calendar.Components[i].(type) {
-		case *VAlarm:
+		if alarm, ok := calendar.Components[i].(*VAlarm); ok {
```

Comment:
I know it's smaller but I'm really fond of type switches. :P

Reply:
Okay, let's keep them.

Reply:
Thanks. :)